src/tools/tidy/src/error_codes_check.rs

```diff
@@ -22,7 +22,7 @@ const IGNORE_EXPLANATION_CHECK: &[&str] = &["E0464", "E0570", "E0601", "E0602",
 //
 // We need to declare each path in the windows version (with backslash).
 const PATHS_TO_IGNORE_FOR_EXTRACTION: &[&str] =
-    &["tests/", "src\\test\\", "src/doc/", "src\\doc\\", "src/tools/", "src\\tools\\"];
+    &["tests/", "tests\\", "src/doc/", "src\\doc\\", "src/tools/", "src\\tools\\"];
```

Comment:
this looks *super* sketchy, but not something you need to fix in this PR

Reply:
Then let's do this in another PR.